File: Python/Pemilik.py
class Pemilik:
    """Kelas untuk merepresentasikan Pemilik Hewan."""
    def __init__(self, nama: str = "", alamat: str = "", no_telepon: str = ""):
        # Atribut private
        self.__nama = nama
        self.__alamat = alamat
        self.__no_telepon = no_telepon

    # Getter konvensional
    def get_nama(self) -> str:
        return self.__nama

    def get_alamat(self) -> str:
        return self.__alamat

    def get_no_telepon(self) -> str:
        return self.__no_telepon

    # Setter konvensional
    def set_nama(self, nama: str):
        self.__nama = nama

    def set_alamat(self, alamat: str):
        self.__alamat = alamat

    def set_no_telepon(self, no_telepon: str):
        self.__no_telepon = no_telepon

    def get_info_singkat(self) -> str:
        """Mengembalikan informasi pemilik untuk ditampilkan di tabel."""
        return f"{self.__nama} ({self.__no_telepon})"

File: Java/Pemilik.java
public class Pemilik {
    // Atribut private
    private String nama;
    private String alamat;
    private String noTelepon;

    // Konstruktor
    public Pemilik(String nama, String alamat, String noTelepon) {
        this.nama = nama;
        this.alamat = alamat;
        this.noTelepon = noTelepon;
    }

    // Konstruktor default
    public Pemilik() {
        this("", "", "");
    }

    // Getter konvensional
    public String getNama() {
        return nama;
    }

    public String getAlamat() {
        return alamat;
    }

    public String getNoTelepon() {
        return noTelepon;
    }

    // Setter konvensional
    public void setNama(String nama) {
        this.nama = nama;
    }

    public void setAlamat(String alamat) {
        this.alamat = alamat;
    }

    public void setNoTelepon(String noTelepon) {
        this.noTelepon = noTelepon;
    }

    //Mengembalikan informasi pemilik untuk ditampilkan di tabel.

    public String getInfoSingkat() {
        return nama + " (" + noTelepon + ")";
    }
}